tianshou/policy/modelbase/psrl.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
+from typing import Any, Dict, Union, Optional
+
+from tianshou.data import Batch
+from tianshou.policy import BasePolicy
+
+
+class PSRLModel(object):
+    """Implementation of Posterior Sampling Reinforcement Learning Model.
+
+    :param np.ndarray trans_count_prior: dirichlet prior (alphas), with shape
+        (n_state, n_action, n_state).
+    :param np.ndarray rew_mean_prior: means of the normal priors of rewards,
+        with shape (n_state, n_action).
+    :param np.ndarray rew_std_prior: standard deviations of the normal priors
+        of rewards, with shape (n_state, n_action).
```

Comment:
I think "q_mean_prior" and "q_std_prior" are better. Rewards of each (s, a) pair are indeed q values.

Reply:
No, Q(s, a) includes discounted return